Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,11 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  default_formula "libfuse"
```

Comment:
Ok. That will cause issues building FUSE formulae on CI, but since nothing in core has a hard dependency on it that shouldn't come up any time soon, so IMO that's not a problem.

This will require a change to the `satisfy` block too, to check for formula-unavailable errors.